src/tiler/sql/user_reservations.sql

```diff
@@ -1,12 +1,12 @@
 (SELECT
   <% if (is_utf_grid) { %>
-  ST_AsGeoJSON(block.geom) AS geojson,
+  CASE WHEN reservation.user_id = <%= user_id %> THEN ST_AsGeoJSON(block.geom) ELSE NULL END AS geojson,
   <% } %>
-  block.geom, block.id, 'reserved' as survey_type
+  block.geom, block.id, CASE WHEN reservation.user_id = <%= user_id %> THEN  'reserved' ELSE 'unavailable' END as survey_type
   FROM survey_blockface AS block
-  INNER JOIN survey_blockfacereservation AS reservation
+  LEFT OUTER JOIN survey_blockfacereservation AS reservation
     ON (block.id = reservation.blockface_id
         AND reservation.canceled_at IS NULL
+        AND reservation.user_id = <%= user_id %>
         AND reservation.expires_at > now() at time zone 'utc')
-  WHERE reservation.user_id = <%= user_id %>
```

Comment:
Since there is no interactivity for `unavailable` blockfaces, it would make sense to me to keep this `WHERE` clause in place when `is_utf_grid` is true.

That would prevent a "hand" pointer over top of blocks that you can't actually click on, and greatly slim down the grid size.
Alternatively, we could show some toast in the UI when you click on an unavailable blockface (but I'd prefer the first option much more).